libraries/core_libs/network/include/network/tarcap/packets_handlers/vote_packet_handler.hpp

```diff
@@ -13,7 +13,8 @@ class VotePacketHandler final : public ExtVotesPacketHandler {
  public:
   VotePacketHandler(std::shared_ptr<PeersState> peers_state, std::shared_ptr<PacketsStats> packets_stats,
                     std::shared_ptr<PbftManager> pbft_mgr, std::shared_ptr<PbftChain> pbft_chain,
-                    std::shared_ptr<VoteManager> vote_mgr, const uint32_t dpos_delay, const addr_t& node_addr);
+                    std::shared_ptr<VoteManager> vote_mgr, const uint32_t dpos_delay,
+                    const uint16_t vote_accepting_rounds, const uint16_t vote_accepting_steps, const addr_t& node_addr);
```

Comment:
just an optional note: if there is more than 1 pram from config that you need to pass here, you could simply pass the whole config instead of 3 separate params...